x/bank/types/send_authorization.go

```diff
@@ -21,26 +19,25 @@ func NewSendAuthorization(spendLimit sdk.Coins) *SendAuthorization {
 
 // MethodName implements Authorization.MethodName.
 func (authorization SendAuthorization) MethodName() string {
```

Comment:
I think we're changing this to `MessageName` in a separate PR right? In previous version this I believe actually returned `sdk.Msg` instead of `string`. Anyway, out of scope from this PR but let's make sure we address in the audit follow-ups.

Reply:
Yes, I added a note on https://github.com/cosmos/cosmos-sdk/issues/9114#issuecomment-823982059. Will follow-up on robert's PR to double check this.